### DISCUSSION TITLE
Any documentation on using CLI?

### DISCUSSION THREAD

**Discussion by shanemcguffin** [OpenRCT2/OpenRCT2#17058] [Q&A] [answered]
Looking for examples of how to create rides, etc.. 

Also, is there any way to to print debug info when using GUI?

**Comment by IntelOrca** [comment, marked as answer]
If you are referring to the scripting / plugin engine, then this is the best reference:
https://github.com/OpenRCT2/OpenRCT2/blob/develop/distribution/openrct2.d.ts

This comes with the game, along with a scripting readme.
